docs/workflows/best-practices-objectives.md

```diff
@@ -54,18 +52,19 @@ To be sure, setting objectives for a single system has value. But in most situat
 - A limited amount of people needs to be involved to make decisions
 - The selected systems support a limited, coherent set of business processes and/or a limited/coherent set of organizational goals.
 
-Typically, divisions or departments are a good starting point (although in large organizations, these might still be quite big). A technology platform (e.g. a low code platform) can be a useful scope because they often exist to accelerate development to decrease the time to market, which is a useful goal to base objectives on. The downside is that they typically support a variety of business processes which makes that less usable as a source of a limited, coherent set of organizational goals.
+Typically, divisions or departments are a good starting point (although in large organizations, these might still be quite big). A technology platform (e.g. a low code platform) can be a useful scope because these platforms often exist to accelerate development to decrease the time to market, which is a useful goal to base objectives on. The downside is that they typically support a variety of business processes which makes that less usable as a source of a limited, coherent set of organizational goals.
 
 ### 2. Establish governance structure
 It needs to be clear who gets to decide what. Our [guidelines on organization](best-practices-organization.md) describe the different stakeholder groups that need to be involved. They all need to be involved somehow:
 - Teams need to understand what is expected of them and provide input on the objectives.
 - Engineering leads take the lead in setting technical standards and indicating their organization value.
-- Management provides organization goals and needs to understand the organization value.
-Note that this does not imply that there has to be a one-person-per-group setup. Consider the SIG situation: Formally, our objective-setting is part of our ISMS (information security management system). Our CISO (Chief Information Security Officer) represents management and is responsible for overall outcomes. Our Head of Development owns the software development process and measurements (objectives) as an engineering lead. Our tech leads are approvers of the process document and represent the teams. Other stakeholders (Head of Product, team leads) are consulted but are not part of the formal process.
+- User representatives provide organization goals needs to understand the organization value of the objectives.
+- Management provides organization goals and constraints not covered by the user representatives.
```

Comment:
It attempts to point out the Management has a responsibility to keep an eye out for stuff that may be overlooked by the other stakeholders because it's not core to the work at hand. I'd imagine legal risk or green could fall in that category.